javafmllanguage/src/main/java/net/minecraftforge/fml/common/Mod.java

```diff
@@ -84,7 +84,7 @@
         enum Bus {
             /**
              * The main Forge Event Bus.
-             * @see MinecraftForge#EVENT_BUS
+             * See {@code MinecraftForge#EVENT_BUS}
```

Comment:
Should have `<p>` prefix. Though it may be better to add another annotation like `@seeplain` that doesn't resolve the symbol.